bitwarden_license/src/Commercial.Core/SecretsManager/Commands/ServiceAccounts/AutoscaleServiceAccountsCommand.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Bit.Core.Models.Business;
+using Bit.Core.OrganizationFeatures.OrganizationSubscriptionUpdate.Interface;
+using Bit.Core.SecretsManager.Commands.ServiceAccounts.Interfaces;
+
+namespace Bit.Commercial.Core.SecretsManager.Commands.ServiceAccounts;
+
+public class AutoscaleServiceAccountsCommand : IAutoscaleServiceAccountsCommand
+{
+    private readonly IUpdateSecretsManagerSubscriptionCommand _updateSecretsManagerSubscriptionCommand;
+
+    public AutoscaleServiceAccountsCommand(
+        IUpdateSecretsManagerSubscriptionCommand updateSecretsManagerSubscriptionCommand)
+    {
+        _updateSecretsManagerSubscriptionCommand = updateSecretsManagerSubscriptionCommand;
+    }
+
+    public async Task AutoscaleServiceAccountsAsync(Guid organizationId, int serviceAccountsToAdd)
```

Comment:
It's easy to mix up "Autoscale seats" (increasing seats due to autoscaling) and "max autoscale seats" (the autoscale limit). I'd prefer to reserve "autoscale" for the limit (i.e. the value in the database), and not use it to describe increasing seats.

I suggest `AdjustSeatsAsync`, because that describes the actual operation. Autoscaling is just the reason.